_posts/2022-07-13-announcing-enmeshed-v2-requests.md

```diff
@@ -0,0 +1,68 @@
+
+This blog post describes what Requests are in the Enmeshed universe and how they are used to exchange Attributes and establish Relationships.
+
+When we mention "the App" or "the Connector" in this blog post, we mean the official [Enmeshed App]({% link _docs_explore/50-app.md %}) and [Enmeshed Connector]({% link _docs_explore/52-connector.md %}).
+
+## Requests
+
+Requests in Enmeshed always defined a way to exchange structured data. In Enmeshed V1 this was exclusively about Attributes. Requests in V2 will also be able to exchange unstructured data without persisting it to the User's Attributes. One example is a form you can send to a User, which contains some questions in natural language.
+
+In Enmeshed V1 `RelationshipTemplates` and `RequestMails` each defined their own way for exchanging Attributes. Further only the App could process them. When integrating via the Connector you had to manually process all Requests. For V2 we pulled the Request handling [from the User-Experience Layer to the Consumption layer]({% link _docs_explore/01-introduction.md %}/#layers). This enabled us to provide you with an API in the Connector to work with Requests. It also helped us making Request handling more flexible and easier to use.
+
+## Exchanging Requests
+
+The simplest way to exchange Requests is using Messages. But for sending Messages a Relationship is required. To create a Relationship we also wanted the possiblity to exchange Requests with the same structure.
+
+### Relationships and RelationshipTemplates
+
+The flow for establishing a Relationship between the App and another App or Connector has changed significantly. The body of the RelationshipTemplate is now a strict type that can be processed by the App and the Connector. It looks as follows:
+
+```ts
+interface RelationshipTemplateBody {
+    "@type": "RelationshipTemplateBody";
+    title?: string;
+    metadata?: object;
+    onNewRelationship: Request;
+    onExistingRelationship?: Request;
+}
+```
+
+More about the structure of a Request will follow in the V2 documentation.
+
+When the RelationshipTemplate is scanned by the App, the Request (defined by the creator of the Template) is rendered for the User. This is similar to the flow in V1. The App will then automatically create the Relationship when the User accepts the Request.
+
+### Messages
+
+Messages can now be used to exchange the same Requests that are used to enter a Relationship. A Message can now simply contain the content `{ "@type": "Request", ... }` or `{ "@type": "Response", ... }`. The `RequestMail`, `AttributesChangeRequest` and `AttributesShareRequest` types are now deprecated and will be removed in the future. Because the Enmeshed V2 App will stop processing these types we strictly advise against further using them.
```

Comment:
So far, only one Request can be sent with one Message. This is intentional, as with RequestItemGroups and RequestItems, various data or actions can be requested. If there is the need to submit multiple Requests, send multiple Messages.